modules/internal-api.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// clair/master.adoc
+
+:_content-type: CONCEPT
+[id="internal-api"]
+= Internal API endpoints for Clair
+
+Internal API endpoints are embedded in `/api/v1/internal` and are meant for communication between Clair microservices.
+
+[IMPORTANT]
+====
+* If your Clair `config.yaml` file is set to `CLAIR_MODE=combo`, internal API endpoints might not exist.
+* APIs are not formally exposed in Clair's OpenAPI Specification. Further information and usage is dependent on the reader.
```

Comment:
@hdonnay I wonder if we should document the internal API? Maybe add some language about what's supported if we keep it?

Reply:
No, internal APIs are internal. It's undocumented so it can change.

Reply:
I _think_ what this is trying to say is that you can't use a `combo` mode instance in a heterogeneous deployment, which is the case.

Reply:
I can omit the internal API section. No problem.